pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -582,6 +582,13 @@ self: super: {
   # 2022-02-14: Strict upper bound: https://github.com/psibi/streamly-bytestring/issues/30
   streamly-bytestring = dontCheck (doJailbreak super.streamly-bytestring);
 
+  streamly-core = appendPatch (pkgs.fetchpatch {
+    url = "https://github.com/composewell/streamly/commit/437c6332a19580a1fe085d593f6bb60b3d9608f8.patch";
+    hash = "sha256-z5v3MGLq2ELlTEGxfHcg2igAM2PC7whx6KIv1Uj2ujw=";
+    includes = ["src/*" "streamly-core.cabal"];
+    stripLen = 1;
+  }) (doJailbreak super.streamly-core) ;
```

Comment:
Could you leave a comment about why this patch is needed?  And ideally when you'd expect it can be removed.